packaging/docker/Dockerfile

```diff
@@ -7,14 +7,8 @@ ARG GOLANG_VERSION
 ################################################################################
 FROM golang:${GOLANG_VERSION} AS builder
 WORKDIR /src
-COPY go.mod go.sum /src/
+COPY . .
```

Comment:
What is the build context that is copied here?

Reply:
Q: why are we not copying go.mod and go.sum first, such that line 10 and line 11 are docker layer cache friendly, then copy the entire directory after that? How is this line related to the dirty state?